inference-engine/src/transformations/src/transformations/common_optimizations/broadcast_elementwise_fusion.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/broadcast_elementwise_fusion.hpp"
+
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::BroadcastElementwiseFusion, "BroadcastElementwiseFusion", 0);
+
+bool is_broadcastable_shapes(const ngraph::PartialShape & input_shape1, const ngraph::PartialShape & input_shape2) {
```

Comment:
There are no tests for this function. We need to create tests for at least all cases provided by @GlebKazantaev in the comment. Also, it make sense to move this function to some common place.

@GlebKazantaev , can you recommend the place where to move this function?

Reply:
In this case we have specific logic which which detect can we eliminate broadcast or not. It is no common case. So we no need move it.

Reply:
Agreed internally to leave the function in this file for now but then move it to utils.